### PR TITLE
Add change log for 1.6.0-rc1

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,141 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add sudo functionality to the API (#4809)
+  * Add oauth providers via cli (#4591)
+  * Disable merging a WIP Pull request (#4529)
+  * Force user to change password (#4489)
+  * Add letsencrypt to Gitea (#4189)
+  * Add push webhook support for mirrored repositories (#4127)
+  * Add csv file render support defaultly (#4105)
+  * Add Recaptcha functionality to Gitea (#4044)
+* BUGFIXES
+  * fix release creation via API (#5076)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Change `f` in `fix` to be uppercase

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,141 @@ This changelog goes through all the changes that have been made in each release
...
+  * Remove links from topics in edit mode  (#5026)
+  * Fix missing AppSubUrl in few more templates (fixup) (#5021)
+  * Fix missing AppSubUrl in some templates (#5020)
+  * Hide outdated comments in file view (#5017)
+  * Upgrade gopkg.in/testfixtures.v2 (#4999)
+  * Disable debug routes unless PPROF is enabled in configuration (#4995)
+  * Fix user menu item styling (#4985)
+  * Fix layout of the topics editing form (#4971)
+  * Fix null pointer dereference in ParseCommitWithSignature (#4962)
+  * fix url in discord webhook (#4953)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Change `f` in `fix` to be uppercase

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,141 @@ This changelog goes through all the changes that have been made in each release
...
+  * Detect charset and convert non UTF-8 files for display (#4950)
+  * Make sure to catch the right error so it is displayed on the UI (#4945)
+  * fix(topics): don't redirect to explore page. (#4938)
+  * fix bug forget to remove Stopwatch when remove repository (#4928)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Change `f` in `fix` to be uppercase

**Review by techknowlogick — CHANGES_REQUESTED** [review]
Some of the items have lower case words for the start of the sentence, in most case they should start with uppercase.

**Comment by lunny** [comment]
@techknowlogick done.

**Review by techknowlogick — APPROVED** [review]

**Review by appleboy — APPROVED** [review]

**Pull request merged**